src/classes/PanelData.js

```diff
@@ -264,8 +264,9 @@ class PanelData {
 	_getCurrentAccount() {
 		const currentAccount = conf.account;
 		if (currentAccount && currentAccount.user) {
-			currentAccount.user.subscriptionsPlus = account.hasScopesUnverified(['subscriptions:plus']);
-			currentAccount.user.subscriptionsPremium = account.hasScopesUnverified(['subscriptions:premium']);
+			currentAccount.user.plusAccess = account.hasScopesUnverified(['subscriptions:plus'])
+													|| account.hasScopesUnverified(['subscriptions:premium']);
+			currentAccount.user.premiumAccess = account.hasScopesUnverified(['subscriptions:premium']);
```

Comment:
I still wanna move this into `Account` but let's worry about that after. We can sneak it into one of the related tickets once this is in testing.

Reply:
I'll make a note!